service/src/main/kotlin/fi/espoo/evaka/pairing/PairingsController.kt

```diff
@@ -122,8 +126,6 @@ class PairingsController(
 
         db.transaction { tx ->
             validatePairing(tx, id, body.challengeKey, body.responseKey)
-
-            // todo: insert into mobile_device and employee ?
```

Comment:
One thing we're still missing is a link between a pairing and the mobile device created from it. Once the pairing process is done, the mobile-side frontend has only a pairing ID + the keys that are sent to the apigw which needs the mobile device ID so it can be stored to the session. I think this validation endpoint could return the mobile device ID, but we also need to have a link in the database to be able to find it. Maybe a nullable `mobile_device_id` to the pairing table?